src/devices/Ccs811/samples/Ccs811.Sample.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Ccs811.csproj" />
+    <ProjectReference Include="..\..\Ft4222\Ft4222.csproj" />
```

Comment:
you might want to simplify the sample for a more common case (or have two samples: for Ft4222 and using protocol directly)

Reply:
actually nvm, you do give users choice 😄 (reviewed this file before the sample)